docs/source/algorithms/ReflectometryReductionOneAuto-v2.rst

```diff
@@ -21,7 +21,7 @@ detectors of interest. Only the detectors of interest will be corrected, the res
 will remain in the original position. Note that when :literal:`ProcessingInstructions` is not set, its value
 is inferred from other properties, depending on the value of :literal:`AnalysisMode`:
```

Comment:
Can we add a brief sentence when we mention algorithm `SpecularReflectionPositionCorrect` saying that detectors will be corrected according to the new property? The reason I suggest this is because I know most of the times scientists only look at the documentation of `ReflectometryReductionOneAuto`.